api/storage_service.cc

```diff
@@ -534,11 +534,17 @@ void unset_view_builder(http_context& ctx, routes& r) {
 }
 
 static future<json::json_return_type> describe_ring_as_json(sharded<service::storage_service>& ss, sstring keyspace) {
-    co_return json::json_return_type(stream_range_as_array(co_await ss.local().describe_ring(keyspace), token_range_endpoints_to_json));
+    apilog.info("describe_ring: keyspace={}: starting", keyspace);
+    auto ret = stream_range_as_array(co_await ss.local().describe_ring(keyspace), token_range_endpoints_to_json);
+    apilog.info("describe_ring: keyspace={}: done", keyspace);
```

Comment:
This is way too noisy. And unnecessary, just remove it.

Reply:
ok